### ISSUE TITLE
undefined is not a object(evaluating 'navigator.userAgent.toLowerCase' ) on React Native 0.61.2

### ISSUE THREAD

**Issue by aminosman** [sendbird/SendBird-SDK-JavaScript#125]
Updated to react native 0.61.2 and now I get this error when I simply just import Sendbird like so `import SendBird from 'sendbird'`. 

Added checks around it by patching the min js and can confirm that I dont get the error anymore but now my connection fails.

`return"undefined"!=typeof navigator&&-1<navigator.userAgent.indexOf("MSIE 10")`
was changed to 
`return"undefined"!=typeof navigator&&"undefined"!=typeof navigator.userAgent&&-1<navigator.userAgent.indexOf("MSIE 10")`

**Comment by ryantando** [comment]
I'm having the same issue on react-native 0.60.3.
Maybe the error appears on 0.60+

Typo on
`return"undefined"!=typeof navigator&&navigator.userAgentt&&-1<navigator.userAgent.indexOf("MSIE 10")`

To
`return"undefined"!=typeof navigator&&navigator.userAgent&&-1<navigator.userAgent.indexOf("MSIE 10")`

**Comment by aminosman** [comment]
Good catch @ryantando, yeah 0.60 added a global navigator with a single property which results in the current safety check not being sufficient 

https://github.com/facebook/react-native/blob/aee88b6843cea63d6aa0b5879ad6ef9da4701846/Libraries/Core/setUpNavigator.js
